pkg/persistentcache/persistentcache.go

```diff
@@ -0,0 +1,57 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2019 Datadog, Inc.
+
+package persistentcache
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/DataDog/datadog-agent/pkg/config"
+)
+
+func getFileForKey(key string) (string, error) {
+	paths := strings.SplitN(key, ":", 2)
```

Comment:
Please add a comment explaining what the key is and why use  `:` as a delimiter for the file name and not `filepath.Dir` and `filepath.Base` ?

Reply:
Done.